node/src/types/node_config.rs

```diff
@@ -33,6 +34,9 @@ pub struct NodeConfig {
 
     /// Flag which forces the node to resync all of the blocks.
     pub force_resync: bool,
+
+    /// Maximum time a node will wait for an upgrade to commit
```

Comment:
Nit:

```suggestion
    /// Maximum time a node will wait for an upgrade to commit.
```

Reply:
Fixed in https://github.com/casper-network/casper-node/pull/3971/commits/2b4c9ae04b1ad1c97b873fabb58367cc72e32144